_includes/color-matrix.html

```diff
@@ -9,7 +9,7 @@
   <svg><use xlink:href="#usa-matrix-bad-contrast-ratio"/></svg>
 
   <p class="usa-sr-invisible" aria-hidden="true">
-    <strong>Don't</strong> use these color combinations. They don’t have enough contrast to meet accessibility standards, and some people will have difficulty reading the text.
+    <strong>Don’t</strong> use these color combinations. They don’t have enough contrast to meet accessibility standards, and some people will have difficulty reading the text.
```

Comment:
@nicoleslaw it's my understanding that for smart quotes like this, we should use HTML entities, which would be `&rsquo;`

My favorite cheatsheet for this is [here](https://www.typewolf.com/cheatsheet#quotes-and-apostrophes)